.circleci/config.yml

```diff
@@ -23,14 +23,18 @@ jobs:
       browser:
         type: string
     executor: linux
+    environment:
+      NODE_ENV: development # Needed for dist folder
     steps:
       - checkout
       - restore_cache:
-          key: deps-{{ .Branch }}--<< parameters.node-version >>--{{ checksum "package.json" }}
+          key: deps-{{ .Branch }}--<< parameters.node-version >>--{{ checksum "package.json" }}--{{ checksum "karma.conf.js" }}
       - node/install:
-          install-npm: false
+          install-npm: false #Node 10 cannot function with `install-npm`
           node-version: << parameters.node-version >>
       - run: npm install
+      - run: ls -la
```

Comment:
@unlikelyzero I had issues again with the `dist` directory missing, which I assume is because `npm run prepare` didn't fire?

Reply:
@scottbell let's get this one merged into master and then you can pull it in https://github.com/nasa/openmct/pull/4566